lib/iterator.js

```diff
@@ -199,6 +199,68 @@ class Iterator {
     return result + suffix;
   }
 
+  // Find value in this iterator compared by `comparator` on values
+  // obtained by applying `accessor` to each value.
```

Comment:
I'd prefer to have them since they are supposed to be a complete sentence, besides those few functions with comments in Iterator mostly have dots at the end.